rbac/management/role/serializer.py

```diff
@@ -139,7 +139,6 @@ def create(self, validated_data):
         role = Role.objects.create(name=name, description=description, display_name=display_name, tenant=tenant)
         create_access_for_role(role, access_list, tenant)
 
-        role_obj_change_notification_handler(role, "created", self.context["request"].user)
```

Comment:
This was moved to `perform_` hooks just for consistency. It has to be there for delete (because delete doesn't use the serializer), and we already have "outbox-y" style things there now (so auditing, outbox, and notifications are all together).